pyccl/halos/profiles.py

```diff
@@ -409,18 +417,19 @@ class HaloProfileGaussian(HaloProfile):
     Args:
         r_scale (:obj:`function`): the width of the profile.
             The signature of this function should be
-            `f(cosmo, M, a, mdef)`, where `cosmo` is a
+            `f(cosmo, M, a, mass_def)`, where `cosmo` is a
             :class:`~pyccl.core.Cosmology` object, `M` is a halo mass in
-            units of M_sun, `a` is the scale factor and `mdef`
+            units of M_sun, `a` is the scale factor and `mass_def`
             is a :class:`~pyccl.halos.massdef.MassDef` object.
         rho0 (:obj:`function`): the amplitude of the profile.
             It should have the same signature as `r_scale`.
     """
     name = 'Gaussian'
 
-    def __init__(self, r_scale, rho0):
+    @warn_api(pairs=[("r_scale", "r_s")])
```

Comment:
why are you doing this? It seems you're not really renaming it into `r_s`, right?

Reply:
OK. Good catch.